github.sgnl.yaml

```diff
@@ -640,6 +654,579 @@ entities:
+      - name: fullDatabaseId
+        externalId: fullDatabaseId
+        type: Int64
+      - name: includesCreatedEdit
+        externalId: includesCreatedEdit
+        type: Bool
+      - name: isPinned
+        externalId: isPinned
+        type: Bool
+      - name: lastEditedAt
+        externalId: lastEditedAt
+        type: DateTime
+      - name: locked
+        externalId: locked
+        type: Bool
+      - name: number
+        externalId: number
+        type: Int64
+      - name: publishedAt
+        externalId: publishedAt
+        type: DateTime
+      - name: state
+        externalId: state
+        type: String
+      - name: stateReason
+        externalId: stateReason
+        type: String
+      - name: title
+        externalId: title
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+  IssueAssignee:
+    displayName: IssueAssignee
+    externalId: IssueAssignee
+    description: Entity representing the connection between a User entity assigned to a GitHub Issue
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: issueId
+        externalId: issueId
+        type: String
+        indexed: true
+        attributeAlias: assignedIssueId
+      - name: userId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: issueAssigneeId
+      - name: login # remove later
+        externalId: login
+        type: String
+  IssueParticipant:
+    displayName: IssueParticipant
+    externalId: IssueParticipant
+    description: Entity representing the connection between a User entity that is a participant of a GitHub Issue
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: issueId
+        externalId: issueId
+        type: String
+        indexed: true
+        attributeAlias: participantIssueId
+      - name: userId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: issueParticipantId
+      - name: login # remove later
+        externalId: login
+        type: String
+  
+  # https://docs.github.com/en/graphql/reference/objects#label
+  Label:
+    displayName: Label
+    externalId: Label
+    description: Entity representing a GitHub Label
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: labelId
+      - name: color
+        externalId: color
+        type: String
+      - name: createdAt
+        externalId: createdAt
+        type: DateTime
+      - name: isDefault
+        externalId: isDefault
+        type: Bool
+      - name: name
```

Comment:
To index?
"Allow access if `prod-bug` label is present"

Reply:
fixed